### ISSUE TITLE
GUROBI_CMD fails in readsol()

### ISSUE THREAD

**Issue by erikliland** [coin-or/pulp#113]
> Pull request: #112 

During  `pulp.pulpTestAll()`, `self.readsol(tmpSol)` in `class GUROBI_CMD` encounters a `ValueError: too many values to unpack (expected 2)` which causes the test to fail.

The code that fails is:
`name, value  = line.split()`  
in
```
for line in my_file:
    name, value  = line.split()
    values[name] = float(value)
```

The line that is causing the `ValueError` is  `# Objective value = 54`. To comparison is a "normal" line `x 4`.

The key `"Objective value"` does not exist in the `values` dictionary and therefore we can skip this value.

Solution suggestion in `readsol()`:
```
try:
    name, value  = line.split()
    value = value.strip()
except ValueError:
    pass
``

**Comment by xoolive** [comment]
I encountered the same problem today on my Gurobi version. However a succession of try/catch doesn't seem to be the proper approach to tackle the problem.

I would rather recommend the following, though this should be safeguarded by the proper gurobi version 😟 (I don't know how to get the proper version number (except by parsing `gurobi_cl`) and don't know from which version number the header changed.)

```python
    def readsol(self, filename):
        """Read a Gurobi solution file"""
        with open(filename) as my_file:
            try:
                next(my_file) # skip the header line <<< add this line
                next(my_file) # skip the objective value
            except StopIteration:
```
Maybe a totally different approach is wiser (skip the # lines, check one of them stands for "objective value")

Anyway, +1 for the issue found.

**Comment by gplssm** [comment]
Thanks for the workaround proposed in #112 @erikliland 

From my opinion this is just a workaround since it does not tackle the actual problem. A warning is prompted then, saying there is no proper objective value of the solution Gurobi has found.

But thanks anyway, it helped me for now!

**Comment by stumitchell** [comment]
should be fixed now in 1.6.6
